packages/fxa-payments-server/src/components/PlanDetails/index.scss

```diff
@@ -0,0 +1,116 @@
+.plan-details-component {
+  background: #fff;
+  border-radius: 8px;
+  box-shadow: 0 12px 18px 2px
+rgba(34, 0, 51, 0.04), 0 6px 22px 4px rgba(7, 48, 114, 0.12), 0 6px 10px -4px
+rgba(14, 13, 26, 0.12);
+  font-size: 15px;
+  margin: 0 auto;
+  padding: 20px 20px 0;
+  width: 342px;
+
+  .plan-details-header-wrap {
+    display: flex;
+  }
+
+  .plan-details-heading-wrap {
+    display: flex;
+    flex-direction: column;
+    padding-left: 10px;
+    text-align: left;
+
+    .plan-details-product {
+      /* Grey/Grey 50 */
+      color: rgba(12, 12, 13, 0.8);
```

Comment:
Do we need to change the opacity of some of these text elements? Can we just use a hex color of a lighter grey (easier to picture the color & probably reuse with variables)?

Reply:
These colors were all copied from figma, I can check with anthony